src/de/mod.rs

```diff
@@ -1936,6 +1937,9 @@ pub enum PayloadEvent<'a> {
     /// Unescaped character data between `Start` and `End` element,
     /// stored in `<![CDATA[...]]>`.
     CData(BytesCData<'a>),
+    /// Unescaped character data between `Start` and `End` element,
+    /// stored in `<!DOCTYPE ... [...]]>`.
```

Comment:
```suggestion
    /// Document type definition data (DTD) stored in `<!DOCTYPE ...]>`.
```
Actually, I would like if you would synchronize documentation between `Event` and `PayloadEvent` (in it's own commit, do that first)